reference/employer/openapi.yaml

```diff
@@ -2440,6 +2523,50 @@ components:
                   - family
                 description: Type for HSA contribution limit if the benefit is a HSA.
                 nullable: true
+    BenefitFeaturesAndOperations:
+      allOf:
+        - $ref: '#/components/schemas/CompanyBenefit'
```

Comment:
the company benefit model has benefit id, type, description and frequency. i think we want a different model here right?

Reply:
You're right! Let me make a new model. Quick question, are `description` and `frequency` not meant to be returned? We return these for `/benefits/meta` I believe
<img width="437" alt="Screenshot 2023-06-22 at 8 45 19 PM" src="https://github.com/Finch-API/finch-docs/assets/23018297/2c65496f-5d92-43fc-9319-69b80b5504f3">

`type` is the only property from `CompanyBenefit` that I need to omit right?

Reply:
they are meant to be returned. frequency will be an array of the supported frequency enums, and description will be a "human readable" display text for the benefit type. so yeah type is the only property to omit, but in this model we're returning booleans and arrays as the values of the properties vs enums/integers in the company benefit object